client/app/states/services/details/details.html

```diff
@@ -116,6 +130,21 @@
                     <input class="form-control" disabled value="${{ ::vm.service.chargeback.used_cost_sum | number:3 }}"/>
                   </div>
                 </div>
+                <div class="form-group">
```

Comment:
@AllenBW You'll notice the `<div class="form-group">` usage in other places too (minus this PR) and I think could be addressed in a separate PR.
What do you think?